src/fragments/lib/project-setup/flutter/prereq/prereq.mdx

```diff
@@ -1,4 +1,3 @@
 - Install a stable version of [Flutter](https://flutter.dev/docs/get-started/install).
-    - For the stable release, use version 2.10.0 or higher
-    - For the developer preview, use version 3.0.0 or higher
+    - Use version 3.0.0 or higher
```

Comment:
Should be 3.3.0. I think the initial dev-preview was 3.0 but it was raised before the first stable release.

```suggestion
    - Use version 3.3.0 or higher
```